src/box/lua/key_def.c

```diff
@@ -59,8 +59,15 @@ luaT_push_key_def(struct lua_State *L, const struct key_def *key_def)
 			lua_setfield(L, -2, "path");
 		}
 
-		lua_pushboolean(L, key_part_is_nullable(part));
-		lua_setfield(L, -2, "is_nullable");
+		if (key_part_is_nullable(part)) {
+			lua_pushboolean(L, true);
+			lua_setfield(L, -2, "is_nullable");
```

Comment:
Can we avoid changing this behaviour in current patch? I guess it is okay to hide is_nullable option, but could you please move this to a separate patch?

Reply:
fixed